AppCenterData/AppCenterData/MSDataSerializationHelpers.h

```diff
@@ -21,6 +21,6 @@
  *
  * @return `NSString` instance representing the date in ISO 8601 format.
  */
-+ (NSString *)serializeDate:(NSDate *)date;
++ (NSString *)serializeDate:(NSDate *)date NS_SWIFT_NAME(serialize(_:));
```

Comment:
Any reason for not having `date` label here, unlike in `deserialize`?

Reply:
The object is a `date` so it's explicit enough while the `deserialize` take a string so we need the distinction.

Reply:
I would prefer to have it for both just to align the signatures

Reply:
In swift that will be redundant though @jwallra